docs/content/contribute/review-pr.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Review a PR"
+weight: 11
+---
+
+# Review a PR
+
+This page provides guidelines for reviewing Magic Modules pull requests
+
+1. Read the pull request description and the linked issues to understand the context and check if the pull request actually resolves the issues.
```

Comment:
If an issue isn't linked, we should have folks check the issue tracker and add resolved issues if any.

I'm not sure how to write it clearly in this guide, but ideally we'd either have every single PR either:

1. Link to an issue with context
2. Establish clear context itself. For simple things (minor fixups, minor test fixes) this may be the title, but in other cases we'd want details in the description. This would serve to tackle contextless PRs.